### PR TITLE
Improve web ux

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
contains https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/313
requires https://github.com/Drakkar-Software/OctoBot-Services/pull/78
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/420

services logos
image
ticker in currency select
image
ticker filter in pair select
image
all currency images
image
enable currencies
image

### CODE REVIEW

**Review by Herklos — APPROVED** [review]

File: Services/Interfaces/web_interface/constants.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 
 # utility URLs
-CURRENCIES_LIST_URL = "https://api.coincap.io/v2/assets?limit=1000"
+CURRENCIES_LIST_URL = "https://api.coingecko.com/api/v3/coins/list"
```
Comment (Herklos) [inline Services/Interfaces/web_interface/constants.py:19]: :+1:

File: Services/Interfaces/web_interface/models/configuration.py [hunk 2]
```diff
@@ -37,6 +37,9 @@
 import octobot.constants as octobot_constants
 
 NAME_KEY = "name"
+SYMBOL_KEY = "symbol"
+ID_KEY = "id"
+EXCLUDED_CURRENCY_SUBNAME = tuple(("X Long", "X Short"))
```
Comment (Herklos) [inline Services/Interfaces/web_interface/models/configuration.py:42]: There is currencies with this end of name ? :open_mouth:

File: Services/Interfaces/web_interface/models/configuration.py [hunk 3]
```diff
@@ -485,16 +488,24 @@ def get_symbol_list(exchanges):
     return list(set(result))
 
 
+def _is_legit_currency(currency):
+    return not any(sub_name in currency for sub_name in EXCLUDED_CURRENCY_SUBNAME) and len(currency) < 30
```
Comment (Herklos) [inline Services/Interfaces/web_interface/models/configuration.py:492]: Why did you need `len(currency) < 30` ? Couldn't we have a legit currency with len(currency_name) > 30 ?

File: Services/Interfaces/web_interface/models/dashboard.py [hunk 4]
```diff
@@ -64,12 +64,13 @@ def format_trades(dict_trade_history):
             trade_type = trade_side
         if status is not trading_enums.OrderStatus.CANCELED.value or DISPLAY_CANCELLED_TRADES:
             trade_time = dict_trade[trading_enums.ExchangeConstantsOrderColumns.TIMESTAMP.value]
-            trades[trade_time_key].append(
-                timestamp_util.convert_timestamp_to_datetime(trade_time, time_format="%y-%m-%d %H:%M:%S"))
-            trades[trade_price_key].append(dict_trade[trading_enums.ExchangeConstantsOrderColumns.PRICE.value])
-            trades[trade_description_key].append(
-                f"{trade_type.name.replace('_', ' ')}: {dict_trade[trading_enums.ExchangeConstantsOrderColumns.AMOUNT.value]}")
-            trades[trade_order_side_key].append(trade_side.value)
+            if trade_time > trading_constants.MINIMUM_VAL_TRADE_TIME:
```
Comment (Herklos) [inline Services/Interfaces/web_interface/models/dashboard.py:67]: I posted a comment about that in https://github.com/Drakkar-Software/OctoBot-Trading/pull/420

File: Services/Interfaces/web_interface/static/js/common/resources_rendering.js [hunk 5]
```diff
@@ -38,23 +38,99 @@ function fetch_images() {
...
+                    }else{
+                        handleDefaultImage(element, currencyDefaultImage);
+                    }
+                }
+            }
+        }
     });
 }
 
+const currencyLoadingImageName = "loading_currency.svg";
```
Comment (Herklos) [inline Services/Interfaces/web_interface/static/js/common/resources_rendering.js:131]: Is it still the same default currency image ?

File: Services/Services_bases/trading_view_service/trading_view.py [hunk 6]
```diff
@@ -73,6 +73,12 @@ def get_endpoint(self) -> None:
     def get_type(self) -> None:
         return services_constants.CONFIG_TRADING_VIEW
 
+    def get_website_url(self):
+        return "https://www.tradingview.com/"
+
+    def get_logo(self):
+        return "https://in.tradingview.com/static/images/favicon.ico"
```
Comment (Herklos) [inline Services/Services_bases/trading_view_service/trading_view.py:80]: :+1:

File: Services/Interfaces/web_interface/models/configuration.py (hunk 2 above)
Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/models/configuration.py:42]: Yes unfortunatelly, and since we very likely don't want them, I think it's better to filter them. ex "RealToken 18900 Mansfield St Detroit MI"

File: Services/Interfaces/web_interface/models/configuration.py (hunk 3 above)
Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/models/configuration.py:492]: From what I've seen, these currencies are "RealToken 18900 Mansfield St Detroit MI", "0-5x-long-ethereum-classic-token" etc. Do you have an example of a legit >30 char currency name ?

File: Services/Interfaces/web_interface/static/js/common/resources_rendering.js (hunk 5 above)
Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/static/js/common/resources_rendering.js:131]: It's this 
image

File: Services/Interfaces/web_interface/models/configuration.py (hunk 2 above)
Comment (Herklos) [reply Services/Interfaces/web_interface/models/configuration.py:42]: :open_mouth:

File: Services/Interfaces/web_interface/models/configuration.py (hunk 3 above)
Comment (Herklos) [reply Services/Interfaces/web_interface/models/configuration.py:492]: No, I was suprised about that but it seems to be a legit maximum

**Review by Herklos — APPROVED** [review]
Nice :+1:

**Comment by GuillaumeDSM** [comment]
contained in https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/317
